public/app/features/explore/RichHistory/RichHistoryCard.tsx

```diff
@@ -254,7 +255,7 @@ export function RichHistoryCard(props: Props) {
         title={query.comment?.length > 0 ? 'Edit comment' : 'Add comment'}
       />
       <IconButton name="copy" onClick={onCopyQuery} title="Copy query to clipboard" />
-      {!isRemoved && <IconButton name="link" onClick={onCreateLink} title="Copy link to clipboard" />}
+      {!isRemoved && <IconButton name="share-alt" onClick={onCreateLink} title="Copy shortened link to clipboard" />}
```

Comment:
We have agreed with @jessover9000 that we will unify all sharing icons to `share-alt`, as `link` icon is used for split mode.